src/Shared/ProblemDetails/HttpValidationProblemDetailsJsonConverter.cs

```diff
@@ -30,7 +34,8 @@ public static HttpValidationProblemDetails ReadProblemDetails(ref Utf8JsonReader
         {
             if (reader.ValueTextEquals(Errors.EncodedUtf8Bytes))
             {
-                var errors = DeserializeErrors(ref reader, options);
+                var context = new ErrorsJsonContext(options);
```

Comment:
Do we need to create a new `ErrorsJsonContext` instance every time through this loop?

Reply:
Working on this in https://github.com/dotnet/aspnetcore/pull/45646